### ISSUE TITLE
Error! Unable to generate Contract ByteCode and ABI

### ISSUE THREAD

**Issue by letmein2** [FreezyEx/PreSale-Contract#2]
Hey FreezyEx,

I'm trying to deploy your ICO script and keep getting the following -  Error! Unable to generate Contract ByteCode and ABI .

I was wondering if you know what causing this. I've looked online but so far haven't found any fix for this. I'd really like to use this to launch my token.

Thanks

**Comment by FreezyEx** [comment]
https://www.notion.so/Pre-Sale-Contract-78bbe4f84363424dbc7cd1388e78121c

**Comment by letmein2** [comment]
Hey Freezy.  I very much appreciate the help so far. I've got the presale contract deployed but when i go to start the ICO i get the following transaction error - Fail with error 'availableTokens should be &gt; 0 and &lt;= totalSupply' .

I set the following settings on deploy for the rate (5 Trill per bnb) , address, token address, etc.  
5000000000000,0x5B9cED86ecDc2d859D8126A08E429f07d9B15409,0xa43c607c836c250f851a50c385d4d906ea829c66

On startICO I set the following



0 | endDate | uint256 | 1624238552   - (Jun 20 2021 20:22:32)

1 | _minPurchase | uint256 | 100000000000000000   - (equal to .1 ETH.BNB)



Not sure why it's giving me the availableTokens error as supply is 1 quadtrill.

Thanks

Mike
